backend/internal/application/store/store.go

```diff
@@ -214,7 +214,7 @@ func buildApplicationFromResultRow(row map[string]interface{}) (model.Applicatio
 	var redirectURIs []string
 	if row["callback_uris"] != nil {
 		if uris, ok := row["callback_uris"].(string); ok {
-			redirectURIs = utils.ParseStringArray(uris)
+			redirectURIs = utils.ParseStringArray(uris, ",")
```

Comment:
we need to store this more cleaner way, rather having to split with the ','.

Reply:
+1. Shall we address this in a separate effort?
Create an issue to track: https://github.com/asgardeo/thunder/issues/239

Reply:
This issue no longer exists for the application service with the improvement done to store additional app configs as a JSON in the database.